lab3.md

```diff
@@ -438,6 +438,45 @@ You can now type in `faas-cli new --lang node10-express`.
 
 See also: [Function & Template Store](https://github.com/openfaas/store/)
 
+### Variable Substitution in YAML File (optional exercise)
+
+The `.yml` file used to configure the CLI is capable of variable substitution so that you are able to use the same `.yml` file for multiple configurations.
+
+One example of where this can be used is when there are different registries for development and production images. You can use the variable substitution so that local and test environments use the default account, and the CI server can be configured to use the production account.
```

Comment:
`One example of where this can be used`

consider used => useful

Reply:
Good catch, thank you. I've updated this.

I've also added  a note mentioning that all references to the variable will be replaced throughout the file.